keyboards/setta21/keymaps/salicylic/keymap.c

```diff
@@ -0,0 +1,206 @@
+// entirely and just use numbers.
+enum layer_number {
+  _BASE = 0,
+  _ARROW,
+  _MACRO,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  RGB_RST = SAFE_RANGE,
+  SEND_SUM,
+  SEND_AVE,
+  SEND_CIF,
+  SEND_MAX,
+  SEND_MIN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT(
+      //,-----------------------------------|
+           KC_ESC,   KC_F2,  JP_EQL,  KC_DEL,
+      //|--------+--------+--------+--------|
+          KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+      //|--------+--------+--------+--------|
+            KC_P7,   KC_P8,   KC_P9,         
+      //|--------+--------+--------+--------|
+            KC_P4,   KC_P5,   KC_P6, KC_PPLS,
+      //|--------+--------+--------+--------|
+            KC_P1,   KC_P2,   KC_P3,         
+      //|--------+--------+--------+--------|
+LT(_ARROW, KC_P0),LT(_MACRO, KC_PDOT),KC_PENT
+      //`-----------------------------------'
+  ),
+
+  [_ARROW] = LAYOUT(
+      //,-----------------------------------|
+          _______, _______, _______, _______,
+      //|--------+--------+--------+--------|
+          XXXXXXX, _______, _______, _______,
+      //|--------+--------+--------+--------|
+          XXXXXXX,   KC_UP, XXXXXXX,         
+      //|--------+--------+--------+--------|
+          KC_LEFT, KC_DOWN,KC_RIGHT, _______,
+      //|--------+--------+--------+--------|
+          XXXXXXX, KC_DOWN, XXXXXXX,         
+      //|--------+--------+--------+--------|
+       MO(_ARROW),       MO(_MACRO), _______ 
+      //`-----------------------------------'
+  ),
+
+  [_MACRO] = LAYOUT(
+      //,-----------------------------------|
+          _______, _______, _______, _______,
+      //|--------+--------+--------+--------|
+         SEND_MIN,SEND_MAX,SEND_CIF,SEND_AVE,
+      //|--------+--------+--------+--------|
+            KC_F7,   KC_F8,   KC_F9,         
+      //|--------+--------+--------+--------|
+            KC_F4,   KC_F5,   KC_F6,SEND_SUM,
+      //|--------+--------+--------+--------|
+           KC_F11,  KC_F12,   KC_F3,         
+      //|--------+--------+--------+--------|
+          _______,          _______, JP_RPRN 
+      //`-----------------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT( /* Base */
+      //,-----------------------------------|
+          _______, _______, _______, _______,
+      //|--------+--------+--------+--------|
+          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+      //|--------+--------+--------+--------|
+          RGB_SAD, RGB_SAI, XXXXXXX,         
+      //|--------+--------+--------+--------|
+          RGB_HUD, RGB_HUI, XXXXXXX, RGB_TOG,
+      //|--------+--------+--------+--------|
+          RGB_VAD, RGB_VAI, XXXXXXX,         
+      //|--------+--------+--------+--------|
+          _______,          _______, RGB_MOD 
+      //`-----------------------------------'
+  )
+};
+
+
+//A description for expressing the layer position in LED mode.
+uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
Could you replace these? 

```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```